.env

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
I think it's better to add to the repository a `.env.example` that anyone should copy and ignore the actual `.env` to prevent some mistakes

Reply:
In that case Taiga won't start by default. Do we want't a further user step to copy to the .env file?

Reply:
I'd like it, but we can try with the default system.